src/app/components/post-item/post-item.component.html

```diff
@@ -27,7 +27,12 @@
     </p>
     <div class="item-card__authors">
       @for (author of post.authors; track $index) {
-        <blog-author [author]="author" [muted]="true"></blog-author>
+        <a
+          [attr.data-role]="$any(author).url ? 'button' : ''"
+          [routerLink]="'/' + $any(author).url"
+          (click)="$event.stopPropagation()">
+          <blog-author [author]="author" [muted]="true"></blog-author>
+        </a>
```

Comment:
The whole card is a link to the post, so we don't need a link to the author here. Please keep `blog-author` component only, without the link tag